api/src/main/java/org/openmrs/module/ugandaemrreports/library/CommonDimensionLibrary.java

```diff
@@ -869,4 +869,123 @@ public CohortDefinitionDimension getPatientTypeDimension(){
 
         return patientTypeDimension;
     }
+
+    public CohortDefinitionDimension getCovidAgeDimension() {
+        CohortDefinitionDimension ageGenderDimension = new CohortDefinitionDimension();
+
+        CohortDefinition below1Year = cohortDefinitionLibrary.below1Year();
+
+        CohortDefinition between1And4Years = cohortDefinitionLibrary.between1And4years();
+        CohortDefinition between5And9Years = cohortDefinitionLibrary.between5And9years();
+
+        CohortDefinition between10And11Years = cohortDefinitionLibrary.between10And11years();
+        CohortDefinition between12And14Years = cohortDefinitionLibrary.between12And14years();
+
+        CohortDefinition between15And17Years = cohortDefinitionLibrary.between15And17years();
+        CohortDefinition between18And19Years = cohortDefinitionLibrary.between18And19years();
+
+        CohortDefinition between20And24Years = cohortDefinitionLibrary.between20And24years();
+        CohortDefinition between25And29Years =cohortDefinitionLibrary.between25And29years();
+
+        CohortDefinition between30And34Years =cohortDefinitionLibrary.between30And34years();
+        CohortDefinition between35And39Years =cohortDefinitionLibrary.between35And39years();
+
+        CohortDefinition between40And44Years =cohortDefinitionLibrary.between40And44years();
+        CohortDefinition between45And49Years =cohortDefinitionLibrary.between45And49years();
+
+        CohortDefinition between50And54years =cohortDefinitionLibrary.between50And54years();
+        CohortDefinition between55And59years =cohortDefinitionLibrary.between55And59years();
+
+        CohortDefinition above60Years = cohortDefinitionLibrary.agedAtLeast(60);
+
+        CohortDefinition males = cohortDefinitionLibrary.males();
+        CohortDefinition females = cohortDefinitionLibrary.females();
+
+
+        CohortDefinition a = df.getPatientsInAll(below1Year, males);
+        CohortDefinition b = df.getPatientsInAll(below1Year, females);
+
+        CohortDefinition c = df.getPatientsInAll(between1And4Years, males);
+        CohortDefinition d = df.getPatientsInAll(between1And4Years, females);
+
+        CohortDefinition e = df.getPatientsInAll(between5And9Years, males);
```

Comment:
@akileng56 please give these cohort definitions proper names.

Reply:
@solemabrothers This has been updated.